integration_tests/mockApis/prisonerSearch.ts

```diff
@@ -3,14 +3,27 @@ import { stubFor } from './wiremock'
 import { Prisoner } from '../../server/data/prisonerOffenderSearchTypes'
 
 export default {
+  stubPrisoner: (prisoner: Prisoner): SuperAgentRequest => {
+    return stubFor({
+      request: {
+        method: 'GET',
+        url: `/offenderSearch/prison/HEI/prisoners?term=${prisoner.prisonerNumber}`,
```

Comment:
Could pass `prisonId` as a parameter here (like in `stubPrisoners`) so it's more obvious what's going on when it's called and for more flexibility. Fine for this PR though - just a suggestion for when this next looked at...